core/extension/gdextension_interface.h

```diff
@@ -381,6 +381,18 @@ typedef struct {
 	GDExtensionVariantPtr *default_arguments;
 } GDExtensionClassMethodInfo;
 
+typedef struct {
+	GDExtensionStringNamePtr name;
+	uint32_t method_flags; // Bitfield of `GDExtensionClassMethodFlags`.
+
+	GDExtensionPropertyInfo return_value;
+	GDExtensionClassMethodArgumentMetadata return_value_metadata;
+
+	uint32_t argument_count;
+	GDExtensionPropertyInfo *arguments_info;
```

Comment:
I would name this `arguments` -- would be consistent with `return_value` and also `GDExtensionMethodInfo::arguments`: 

https://github.com/godotengine/godot/blob/a0106607e889fb0ed4ab5bf84a4b2707c3ac5598/core/extension/gdextension_interface.h#L242-L255

Reply:
I used `arguments_info` to be consistent with `GDExtensionClassMethodInfo::arguments_info` which is (in my opinion) a more similar struct to this new one than `GDExtensionMethodInfo`. It's unfortunate that we already have these inconsistencies in the API. :-/ Given that, I'm not sure what the right choice is here.

Reply:
Ah, I see. Well, then we have to pick our poison 😀 I'm fine with keeping it more consistent with the other one.

I'm not sure if this is somehow worth documenting, in case the question comes up again? Maybe not 🤔

Reply:
Ok, let's stick with this name in this case. Probably the best place to document this is on the big list of "things to change when we can completely break compatibility" :-)

Reply:
Do we have such a list yet? 😀